src/rp2_common/hardware_clocks/include/hardware/clocks.h

```diff
@@ -354,9 +354,26 @@ void clocks_enable_resus(resus_callback_t resus_callback);
  * \param gpio The GPIO pin to output the clock to. Valid GPIOs are: 21, 23, 24, 25. These GPIOs are connected to the GPOUT0-3 clock generators.
```

Comment:
I see that there's a
```c
#if !PICO_RP2040
    else if (gpio == 13) gpclk = clk_gpout0;
    else if (gpio == 15) gpclk = clk_gpout1;
#endif
```
in the code, so do we need one of those `\if rp2350_specific` things here?
(this is slightly tangential to the focus of this PR, sorry)

Reply:
I have added some text without the \if because I'm not sure how parameter docs cope with that - side quest for someoine